ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -589,9 +521,7 @@ def get_disk_safety_buckets(logger):
                     min_disk_safety = bucket[0]
                     max_disk_safety = bucket[1]
                     calculated_disk_safety = bucket_safety['remaining_safety']
-                    to_be_added_namespace = {'namespace': namespace['namespace'], 'amount_in_bucket':
-                                                                                  (bucket_safety['count'] /
-                                                                                   namespace['safety_count'])*100}
+                    to_be_added_namespace = {'namespace': namespace['namespace'], 'amount_in_bucket': (bucket_safety['count'] / namespace['safety_count'])*100}
                     if calculated_disk_safety in disk_safety_overview[alba_backend.name][str(min_disk_safety)+','+str(max_disk_safety)]['current_disk_safety']:
                         disk_safety_overview[alba_backend.name][str(min_disk_safety) + ',' + str(max_disk_safety)][
```

Comment:
Put the `current_disk_safety` in a separate value to decrease the dict inception please